spec/features/saml/multiple_endpoints_spec.rb

```diff
@@ -81,17 +81,30 @@
       expect(cert_base64).to eq(Base64.strict_encode64(endpoint_cert.to_der))
     end
 
-    it 'includes the correct auth url, and no SingleLogoutService urls' do
+    it 'includes the correct auth url' do
       visit endpoint_metadata_path
       document = REXML::Document.new(page.html)
       auth_node = REXML::XPath.first(document, '//SingleSignOnService')
-      logout_node = REXML::XPath.first(document, '//SingleLogoutService')
 
       expect(auth_node.attributes['Location']).to include(
         ['/api/saml/auth', endpoint_suffix].join(''),
       )
+    end
 
-      expect(logout_node).to be_nil
+    it 'includes the front-channel logout url' do
+      visit endpoint_metadata_path
+      document = REXML::Document.new(page.html)
+      logout_nodes = REXML::XPath.match(document, '//SingleLogoutService')
+      expect(logout_nodes.count { |n| n['Location'].match?(%r{/api/saml/logout\d{4}}) }).to \
+        eq(2)
```

Comment:
stylewise, I'd break the line a little differently so we can avoid the slash
```suggestion
      expect(logout_nodes.count { |n| n['Location'].match?(%r{/api/saml/logout\d{4}}) }).
        to eq(2)
```

Reply:
🤦 that's much better, still not used to the trailing period